index.html

```diff
@@ -17,6 +17,7 @@
         visibility: visible;
       }
     </style>
+    <link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/tokens/1.0.0-alpha.9/finn-no.css" />
```

Comment:
This should use an Eik alias so it doesn't need constant updates

Reply:
Yup! 👍 Don't think that there was an alias... and not sure whether this link was the way to go forward, so just included it to get started. 😄

Reply:
Alias in place now, so I updated this. 👍